elements/pl-drawing/pl-drawing.py

```diff
@@ -1863,6 +1864,7 @@ def grade(element_html, data):
     grid_size = pl.get_integer_attrib(element, 'grid-size', element_defaults['grid-size'])
     tol = pl.get_float_attrib(element, 'tol', grid_size / 2)
     angtol = pl.get_float_attrib(element, 'angle-tol', element_defaults['angle-tol'])
+    disregard_extra = pl.get_boolean_attrib(element, 'disregard-extra-elements', element_defaults['disregard-extra-elements'])
```

Comment:
We've been trying to keep the python variable names in elements identical to the attribute names (except `-` -> `_`)